pkg/server/observer.go

```diff
@@ -289,124 +289,166 @@ func (s *ObserverServer) GetFlows(
 	req *observer.GetFlowsRequest,
 	server observer.Observer_GetFlowsServer,
 ) (err error) {
-	return getFlowsFromObserver(req, server, s)
+	return getFlows(req, server, s)
 }
 
-func getFlowsFromObserver(
+func getFlows(
 	req *observer.GetFlowsRequest,
 	server observer.Observer_GetFlowsServer,
 	obs Observer,
 ) (err error) {
-	reply, err := getFlows(server.Context(), obs.GetLogger(), obs.GetRingBuffer(), req)
+	start := time.Now()
+	log := obs.GetLogger()
+	ring := obs.GetRingBuffer()
+
+	i := uint64(0)
+	defer func() {
+		size := ring.Cap()
+		took := time.Now().Sub(start)
```

Comment:
can inline these into the log, no need for new vars

Reply:
This code is as it was but I can sure inline this.